README.md

```diff
@@ -1,6 +1,8 @@
 # orchestrator-helm-operator
 Meta Operator for deploying the Orchestrator helm charts
 
+# Pre-install requirements
+This operator is a helm operator using the helm charts from the orchestrator repository. As such, you need to fullfil the pre-install requirements defined in the [README.md](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/README.md) prior to deploying the operator in a cluster.
```

Comment:
Minor spelling - fulfill 

Otherwise lgtm

Reply:
Force pushed the commit.